WcaOnRails/app/controllers/admin_controller.rb

```diff
@@ -481,7 +481,7 @@ def do_complete_persons
 
     ActiveRecord::Base.transaction do
       params[:person_completions].each do |person_key, procedure|
-        next if [:competition_ids, :continue_batch].include? person_key.to_sym
+        next if [:competition_ids, :continue_batch, :clear_inbox_persons_after_finish].include? person_key.to_sym
```

Comment:
I really don't know what is happening here, I did this because `continue_batch` did this. 😅

Reply:
What happens is that the `each` loop above expects every entry in the array to be a "do something with inbox persons" action. The few exceptions mentioned here in the `next if` array are exceptions that should not be processed.